openhands/events/action/commands.py

```diff
@@ -12,19 +12,11 @@
 @dataclass
 class CmdRunAction(Action):
     command: str
-    thought: str = ''
```

Comment:
can you add a test to make sure that existing CmdRunActions (with these fields set) can deserialize properly? Otherwise we'll break existing sessions